src/index.js

```diff
@@ -230,7 +233,7 @@ export default function commonjs ( options = {} ) {
 			var intros = [];
 
 			if ( bundleUsesGlobal ) {
-				intros.push( `var __commonjs_global = typeof window !== 'undefined' ? window : typeof global !== 'undefined' ? global : this;` );
+				intros.push( `var __commonjs_global = typeof window !== 'undefined' ? window : typeof global !== 'undefined' ? global :  typeof self !== 'undefined' ? self : this;` );
```

Comment:
I see, it's or web workers. Can you move it in separate PR also with some tests (is it possible somehow)?